src/i18next.js

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
is this file imported into actual production code?

Reply:
will be, but it will load only one resource for current language